Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs

```diff
@@ -88,6 +92,17 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateImage();
 		}
 
+		void SetControlPropertiesFromProxy()
+		{
+			foreach (UIControlState uiControlState in s_controlStates)
+			{
+				Control.SetTitleColor(UIButton.Appearance.TitleColor(uiControlState), uiControlState); // if new values are null, old values are preserved.
+				Control.SetTitleShadowColor(UIButton.Appearance.TitleShadowColor(uiControlState), uiControlState);
+				Control.SetBackgroundImage(UIButton.Appearance.BackgroundImageForState(uiControlState), uiControlState);
+				Control.SetImage(UIButton.Appearance.ImageForState(uiControlState), uiControlState); // UpdateImage() renders this useless.
```

Comment:
Not sure what went wrong with the image, but I replaced it with an existing one.